pkg/io/mdio/acl.go

```diff
@@ -24,12 +26,12 @@ func makeACLTable(t *ir.ACL, subnet string) [][]string {
 	return rows
 }
 
-func ACLPort(p ir.PortRange) string {
+func ACLPort(p interval.Interval) string {
 	switch {
-	case p.Min == ir.DefaultMinPort && p.Max == ir.DefaultMaxPort:
+	case p.Start() == netp.MinPort && p.End() == netp.MaxPort:
 		return "any port" //nolint:goconst // independent decision for SG and ACL
 	default:
-		return fmt.Sprintf("ports %v-%v", p.Min, p.Max)
+		return fmt.Sprintf("ports %v", p)
```

Comment:
This seems like a breaking change, but we don't have an md test with protocol != netp.AnyProtocol
Please consider adding one.

Reply:
fixed. 
added more tests [here](#205 )